### PR TITLE
KIALI-2328 Fix test failing but ignored and cleanup console errors

### PR DESCRIPTION
**Pull request by israel-hdez**
* Fix failing tests in IstioConfigListComponent.test.ts
* Fix error console.error messages

https://issues.jboss.org/browse/KIALI-2328

`yarn test` should now run beautifully without any warning nor error messages.

### CODE REVIEW

File: src/app/History.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { createBrowserHistory } from 'history';
+import createMemoryHistory from 'history/createMemoryHistory';
 
 const webRoot = (window as any).WEB_ROOT ? (window as any).WEB_ROOT : undefined;
 const baseName = webRoot && webRoot !== '/' ? webRoot + '/console' : '/console';
-const history = createBrowserHistory({ basename: baseName });
+const history = process.env.TEST_RUNNER ? createMemoryHistory() : createBrowserHistory({ basename: baseName });
```
Comment (jotak) [inline src/app/History.ts]: For me, tests are still working without this change... what is it for?

Comment (jotak) [reply src/app/History.ts]: Fine, just saw your comment
> `yarn test` should now run beautifully without any warning nor error messages.

Looks good!

**Comment by israel-hdez** [comment]
@jotak Ops! I missed the last one. It's fixed.

File: src/app/History.ts (hunk 1 above)
Comment (israel-hdez) [reply src/app/History.ts]: To add clarity, this is the kind of errors that this is supressing:
image

They are ignored but, for me, a little bit annoying

**Comment by israel-hdez** [comment]
@jotak Ok, There was a recent change that introduced the test failures.
I rebased and fixed the failing tests.

**Pull request merged**